sapi/fpm/fpm/fpm_main.c

```diff
@@ -600,9 +601,15 @@ void sapi_cgi_log_fastcgi(int level, char *message, size_t len)
 	 * - the message is not empty
 	 * - the fcgi_write did not fail
 	 */
-	if (CGIG(fcgi_logging) && request && message && len > 0
-			&& fcgi_write(request, FCGI_STDERR, message, len) < 0) {
-		php_handle_aborted_connection();
+	if (CGIG(fcgi_logging) && request && message && len > 0) {
```

Comment:
do you think it s worthy to slightly update the comment above ?

Reply:
Yeah that last line wasn't correct anymore...